src/js/service/chargeChange/error.js

```diff
@@ -0,0 +1,14 @@
+import { ERRORS } from '../../constants/index.js';
+
+export const validateAmount = (amount) => {
+  let errorMessage = '';
+
+  const priceLastNumber = Number(amount.toString().charAt(amount.toString().length - 1));
+
+  if (priceLastNumber !== 0) {
+    errorMessage = ERRORS.AMOUNT_UNIT;
+    throw Error(errorMessage);
+  }
```

Comment:
```suggestion
  if (priceLastNumber !== 0) {
    throw Error(ERRORS.AMOUNT_UNIT);
  }
```

바로 값 전달해도 충분히 의미전달이 될 것 같아요